app.js

```diff
@@ -1,9 +1,26 @@
 import express from "express";
+import cors from "cors";
+
+import { AuthorizationController } from "./controllers/authorization.controller.js";
+
+import { checkToken } from "./middleware/token.middleware.js";
+import { WeatherController } from "./controllers/weather.controller.js";
+
+const { CORS_ORIGIN } = process.env;
 
 const app = express();
 
-app.get("/", (req, res) => {
-  res.send("works");
-});
+app.use(
+  cors({
+    credentials: true,
+    origin: CORS_ORIGIN,
+  }),
+);
+
+app.get("/authorization", checkToken, AuthorizationController.authCheck);
+
+app.post("/login", AuthorizationController.login);
```

Comment:
If you decided to implement login endpoint, it would be better if you save some mock user with login and password, and create readme where you will provide it to simulate real login flow instead of just click on the button